Automattic/Automattic-Tracks-Android
# Experimentation without FluxC: example app

Preferably, review:
- #239 
- #240 

first.

### Description

This PR adds possibility to test the `Experimentation` module via the `example` app.

### How to test?

- Go to ExPlat, select a running experiment (preferably with a big number of participants). 
- Enter necessary data: platform to query, experiments name. 
- Generate anon id, tap "setup"
- Test fetching, getting local cache, clearing and see that it works as you'd expect.


<img width="300" alt="Screenshot 2024-08-28 at 09 28 03" src="https://github.com/user-attachments/assets/f8304cb5-1329-44e6-be2d-a4cbab3f3bea">
